src/test/java/org/assertj/core/internal/iterables/Iterables_assertNoneSatisfy_Test.java

```diff
@@ -67,7 +67,18 @@ public void should_fail_when_one_elements_satisfy_the_given_restrictions() {
     // WHEN
     Throwable assertionError = catchThrowable(() -> iterables.assertNoneSatisfy(someInfo(), actual, restrictions));
     // THEN
-    verify(failures).failure(info, noElementsShouldSatisfy(actual, "Yoda"));
+    verify(failures).failure(info, noElementsShouldSatisfy(actual, list("Yoda")));
+    assertThat(assertionError).isNotNull();
+  }
+
+  @Test
+  public void should_fail_when_two_elements_satisfy_the_given_restrictions() {
+    // GIVEN
+    Consumer<String> restrictions = name -> assertThat(name).startsWith("L");
+    // WHEN
+    Throwable assertionError = catchThrowable(() -> iterables.assertNoneSatisfy(someInfo(), actual, restrictions));
```

Comment:
can be replaced by `AssertionsUtil.expectAssertionError` that checks `assertThat(assertionError).isNotNull();` internally.

Reply:
:+1: